recipes/c-blosc2/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conans import ConanFile, CMake, tools
+from conan.tools.microsoft import is_msvc
+import os
+import functools
+
+required_conan_version = ">=1.45.0"
+
+class CBlosc2Conan(ConanFile):
+    name = "c-blosc2"
+    description = "A fast, compressed, persistent binary data store library for C."
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Blosc/c-blosc2"
+    topics = ("c-blosc", "blosc", "compression")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "simd_intrinsics": [None, "sse2", "avx2"],
+        "with_lz4": [True, False],
+        "with_zlib": [None, "zlib", "zlib-ng"],
+        "with_zstd": [True, False],
+        "with_plugins": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "simd_intrinsics": "avx2",
+        "with_lz4": True,
+        "with_zlib": "zlib",
+        "with_zstd": True,
+        "with_plugins": True,
+    }
+    generators = "cmake", "cmake_find_package"
```

Comment:
Upstream overrides CMAKE_MODULE_PATH: https://github.com/Blosc/c-blosc2/blob/v2.2.0/CMakeLists.txt#L123, so it defeats cmake_find_package

Moreover, they rely on custom Find module files: https://github.com/Blosc/c-blosc2/tree/v2.2.0/cmake

Reply:
I think I solved it.
- remove findxxx.cmake in cmake folder
- fix find_package arguments